signer/src/transaction_coordinator.rs

```diff
@@ -540,6 +541,13 @@ where
             )
             .await?;
 
+            metrics::counter!(
+                crate::metrics::TRANSACTIONS_SUBMITTED_TOTAL,
+                "blockchain" => "bitcoin",
+                "status" => "success",
+            )
+            .increment(1);
```

Comment:
nit: we already increment this in `sign_and_broadcast`.

Reply:
🙏🏽